mlflow/pyfunc/mlserver.py

```diff
@@ -1,13 +1,17 @@
 import os
+from mlflow.exceptions import MlflowException, INVALID_PARAMETER_VALUE
 
-from typing import Tuple, Dict
+from typing import Tuple, Dict, Optional
 
 MLServerMLflowRuntime = "mlserver_mlflow.MLflowRuntime"
 MLServerDefaultModelName = "mlflow-model"
 
 
 def get_cmd(
-    model_uri: str, port: int = None, host: str = None, nworkers: int = None
+    model_uri: str,
+    port: Optional[int] = None,
+    host: Optional[str] = None,
+    nworkers: Optional[int] = None,
 ) -> Tuple[str, Dict[str, str]]:
```

Comment:
Does this change relate to this PR?

Reply:
Doesn't seem like it, but it doesn't hurt.